glotaran/project/project.py

```diff
@@ -0,0 +1,444 @@
+"""The glotaran project module."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from os import getcwd
+from pathlib import Path
+from typing import Any
+from typing import Literal
+
+import xarray as xr
+from yaml import safe_load
```

Comment:
`yaml` isn't a builtin module and we changed for `pyyaml` to `ruamel.yaml`  use this instead so the benchmarks won't crash. This only works in the other tests because we get `pyyaml` as an indirect dependency from one of our test dependencies.
`ruamel.yaml` also has a [save loader](https://yaml.readthedocs.io/en/latest/basicuse.html#basic-usage)

Reply:
I resolved from using built in yml and expose the load function.